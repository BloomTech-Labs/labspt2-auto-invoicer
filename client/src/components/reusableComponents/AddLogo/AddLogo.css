.logo {
  border: 1px solid gray;
  background: gray;
  color: white;
  width: 150px;
  height: 50px;
}
