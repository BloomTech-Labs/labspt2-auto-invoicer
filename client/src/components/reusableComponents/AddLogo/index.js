import React from "react";
import "./AddLogo.css";

function AddLogo() {
  return <div className="logo">Add Logo Here</div>;
}

export default AddLogo;
