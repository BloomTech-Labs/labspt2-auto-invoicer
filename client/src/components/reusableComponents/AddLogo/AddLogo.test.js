import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import AddLogo from "../AddLogo";

describe("<AddLogo />", () => {
  it("renders Add Logo", () => {
    const { asFragment } = render(<AddLogo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
