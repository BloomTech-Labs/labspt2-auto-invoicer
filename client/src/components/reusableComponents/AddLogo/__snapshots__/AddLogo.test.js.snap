// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddLogo /> renders Add Logo 1`] = `
<DocumentFragment>
  <div
    class="logo"
  >
    Add Logo Here
  </div>
</DocumentFragment>
`;
